dev/build_arrow.sh

```diff
@@ -70,7 +70,8 @@ function build_arrow_java() {
 
     mvn clean install -pl bom,maven/module-info-compiler-maven-plugin,vector -am \
           -DskipTests -Drat.skip -Dmaven.gitcommitid.skip -Dcheckstyle.skip -Dassembly.skipAssembly
-
+    
+    export CMAKE_BUILD_PARALLEL_LEVEL=112
```

Comment:
better to use `nproc`

Reply:
I guess this variable can impact velox/gluten-cpp build also. If this is the purpose, maybe, just move the `export` to builddeps-velox.sh.

Reply:
velox and gluten-cpp build use their own computed parallel level(also is NPROC), put it here because user may build step by step.